### ISSUE TITLE
"on_join_result" signal not getting correctly fired after join

### ISSUE THREAD

**Issue by Comyune** [alfredbaudisch/GodotPhoenixChannels#24]
I might be _very_ wrong about this, so please school me if I need it! :D

__This signal__:

```gdscript
_error = channel.connect("on_join_result", self, "_on_Channel_join_result")
```

Doesn't seem to be firing at all.
I've been able to work around it by manually triggering from the `on_channel_event` signal.

I believe it has something to do with the `_join_ref`.
I wonder if the join is called multiple times, redefining the `_join_ref` and then it fails to match when the first call responds?

__My network implementation:__

https://github.com/Comyune/game-drifter/blob/b5cc26a024d460ccfd47d0bc351051fb26fcea7e/godot-project/Network.gd

__Here is some context from my live streams of the issue:__

- https://youtu.be/0JiaN38gO7c?t=4692

- https://youtu.be/0JiaN38gO7c?t=4076

- https://youtu.be/W3m7-6FV0KE?t=23485

**Comment by jacopofar** [comment]
I think it's this: https://github.com/alfredbaudisch/GodotPhoenixChannels/pull/26

**Comment by Kyrio** [comment]
Yes I think it's the same issue, I'm waiting for @alfredbaudisch to merge it.

**Comment by alfredbaudisch** [comment]
Fixed #26
